Core/NoOpRunner.Core/SpritesUriHandler.cs

```diff
@@ -3,60 +3,63 @@
 using System.IO;
 using System.Reflection;
 using System.Resources;
+using System.Windows;
+using System.Windows.Automation;
 
 namespace NoOpRunner.Core
 {
     public static class SpritesUriHandler
     {
-        private static readonly string SpritesDirectory = AppDomain.CurrentDomain.BaseDirectory + @"..\..\Resources";
-        private static readonly string SpritesList = "SpritesList.txt";
-
-        private static Dictionary<string, Uri> sprites;
+        private static readonly ResourceReader resourceReader;
 
         /// <summary>
         /// Static but still bad
         /// </summary>
         static SpritesUriHandler()
         {
-            sprites = new Dictionary<string, Uri>();
-
-            var spritesLines = File.ReadAllLines(SpritesDirectory + "\\" + SpritesList);
-
-            foreach (var spriteLine in spritesLines)
-            {
-                sprites.Add(spriteLine.Split('=')[0],
-                    new Uri(SpritesDirectory + "\\Sprites" + spriteLine.Split('=')[1]));
-            }
+            resourceReader = new System.Resources.ResourceReader(@"..\..\Resources\SpritesList.resources");
         }
 
         public static Uri GetPlatformUri()
         {
-            return sprites["platform"];
+            resourceReader.GetResourceData("platform", out string dataType,  out byte[] data);
```

Comment:
Since you don't use the `dataType`, use discards (here and in the rest of the methods). `out string dataType` -> `out _`